toolchain/check/testdata/operators/overloaded/string_indexing_negative.carbon

```diff
@@ -0,0 +1,21 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/full.carbon
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/operators/overloaded/string_indexing_negative.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/operators/overloaded/string_indexing_negative.carbon
+
+
+// --- fail_negative_index.carbon
```

Comment:
Consider moving this test into the string_indexing.carbon file, so we only have to compile the prelude once?